db/migrate/20190517153452_create_rule_references.rb

```diff
@@ -0,0 +1,8 @@
+class CreateRuleReferences < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rule_references, id: :uuid do |t|
```

Comment:
Shouldn't these fields have a validation for presence at least?

Reply:
Wait, you mean like this? https://github.com/RedHatInsights/compliance-backend/pull/146/files#diff-829c4379c551ce7d1b0c9a9e1bbead61R9

Reply:
I was still missing a presence validation on the href, so I've added that.